packages/core/src/code_assist/oauth2.ts

```diff
@@ -275,57 +270,73 @@ export function getCachedGoogleAccountId(): string | null {
   }
 }
 
+function getGoogleAccountEmailCachePath(): string {
+  return path.join(os.homedir(), GEMINI_DIR, GOOGLE_ACCOUNT_EMAIL_FILENAME);
```

Comment:
Should this go in the tmp dir there?

Reply:
I think that the tmp directory manages settings that a different for different projects, since the immediate subdirectories seem to be project-name-hashes(?). Since auth is being maintained globally this setting would need to live somewhere outside tmp. Keeping as-is for now.